forge-gui/res/cardsfolder/upcoming/hustle_bustle.txt

```diff
@@ -0,0 +1,19 @@
+Name:Hustle
+ManaCost:UR
+Types:Instant
+A:SP$ GenericChoice | Defined$ You | Choices$ DBMustAttackEffect,DBMustBlockEffect | ValidTgts$ Creature | SpellDescription$ Target creature attacks or blocks this turn if able.
```

Comment:
I don't think there's actually a choice here

Reply:
or maybe it is ugh

Reply:
hmm how should this be scripted ?

Reply:
just create one effect